<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-05-22T07:46:14 EDT" name="Suite" finished-at="2021-05-22T07:46:16 EDT" duration-ms="1473">
    <groups>
    </groups>
    <test started-at="2021-05-22T07:46:14 EDT" name="Test" finished-at="2021-05-22T07:46:16 EDT" duration-ms="1473">
      <class name="org.restassuredpoc.base.BookingTest">
        <test-method is-config="true" signature="initailization()[pri:0, instance:org.restassuredpoc.base.BookingTest@6236eb5f]" started-at="2021-05-22T07:46:14 EDT" name="initailization" finished-at="2021-05-22T07:46:14 EDT" duration-ms="79" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initailization -->
        <test-method is-config="true" signature="requestPropertySetter()[pri:0, instance:org.restassuredpoc.base.BookingTest@6236eb5f]" started-at="2021-05-22T07:46:14 EDT" name="requestPropertySetter" finished-at="2021-05-22T07:46:16 EDT" duration-ms="1370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- requestPropertySetter -->
        <test-method signature="tsetAuth()[pri:0, instance:org.restassuredpoc.base.BookingTest@6236eb5f]" started-at="2021-05-22T07:46:16 EDT" name="tsetAuth" finished-at="2021-05-22T07:46:16 EDT" duration-ms="15" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
Syntax error, insert "VariableDeclarators" to complete LocalVariableDeclaration
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
Assert cannot be resolved
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error, insert "VariableDeclarators" to complete LocalVariableDeclaration
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
Assert cannot be resolved

at org.restassuredpoc.base.BookingTest.tsetAuth(BookingTest.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tsetAuth -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:org.restassuredpoc.base.BookingTest@6236eb5f]" started-at="2021-05-22T07:46:17 EDT" name="tearDown" finished-at="2021-05-22T07:46:17 EDT" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.restassuredpoc.base.BookingTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
